buzzmobile/constants.yaml

```diff
@@ -44,7 +44,7 @@ wheel_separation: 1.3716 # distance between left and right wheels in meters
 max_steering_angle: 0.262 # radians (=15 degrees)
 
 # Max speed of buzzmobile in m/s
-max_speed: 0.2
+max_speed: 2 #TODO this is a murder machine
```

Comment:
are max speed is likely less than 2 m/s but also more than 0.2. I can update the TODO to be more professional (I'll do that for all of them in this PR), but max speed is something we just need to measure. If we left the max speed at 0.2 the car won't move.